### PR TITLE
*: add Makefile for msc

### PR DESCRIPTION
**Pull request by rleungx**
Signed-off-by: Ryan Leung <rleungx@gmail.com>

### What problem does this PR solve?

Issue Number: Ref #5851.

### What is changed and how does it work?
Let `make check` work for `msc`.

```commit-message
```

### Check List

Tests

- Unit test

### Release note

```release-note
None
```

### CODE REVIEW

**Review by CabinfeverB — APPROVED** [review]

File: tests/client/Makefile [hunk 1]
```diff
@@ -46,6 +46,3 @@ enable-codegen:
 disable-codegen:
 	cd ../../ && $(MAKE) failpoint-disable
 	go mod tidy
-
```
Comment (lhy1024) [inline tests/client/Makefile:49]: why remove it?

**Review by lhy1024 — COMMENTED** [review]
The rest lgtm

**Review by nolouch — APPROVED** [review]

File: tests/client/Makefile (hunk 1 above)
Comment (rleungx) [reply tests/client/Makefile:49]: We don't use it.

**Comment by rleungx** [comment]
I'm going to merge this PR. I have no idea why there are two approve but only get one LGTM label.

**Pull request merged**